src/main/java/org/reaktivity/nukleus/ws/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -214,6 +216,7 @@ private void processBegin(
                 this.targetId = newTargetId;
 
                 this.streamState = this::afterBeginOrData;
+                this.applyInitialWindowAdjustment = true;
```

Comment:
Please use a `windowHandler` field of type `Consumer<WindowFW>`, move the `wrap` at line 335 to line 319, then replace `processWindow` call with `windowHandler.accept(window)`.

Initialize `windowHandler` to `processInitialWindow` which initializes `targetWindowBytesAdjustment` to `80%` of `window.update()`, then sets `windowHandler` to `processWindow` and calls `windowHandler` directly.